src/shared/azureagent/agent.go

```diff
@@ -40,7 +40,7 @@ type Agent struct {
 	managedClustersClient               AzureARMContainerServiceManagedClustersClient
 	userAssignedIdentitiesClient        AzureARMMSIUserAssignedIdentitiesClient
 	federatedIdentityCredentialsClient  AzureARMMSIFederatedIdentityCredentialsClient
-	roleDefinitionsClient               AzureARMAuthorizationRoleDefinitionsClient
+	RoleDefinitionsClient               AzureARMAuthorizationRoleDefinitionsClient
```

Comment:
Why is this public now? As far as I can tell it's not needed and can stay private?

Reply:
IDE lost its mind. Reverted